src/App/App.tsx

```diff
@@ -53,140 +53,142 @@ import './App.css';
 class App extends Component {
   public render() {
     return (
-      <Container>
+      <Container fluid={true}>
         <Helmet titleTemplate={`%s | ` + WEBSITE_NAME} defaultTitle="" />
         <ConnectedHeader />
-        <Row id="main-page-row">
-          <Col>
-            <Switch>
-              <ConnectedPrivateRoute
-                disableLoginProtection={DISABLE_LOGIN_PROTECTION}
-                exact={true}
-                path="/"
-                component={Home}
-              />
-              <ConnectedPrivateRoute
-                disableLoginProtection={DISABLE_LOGIN_PROTECTION}
-                exact={true}
-                path={INTERVENTION_IRS_URL}
-                component={IrsPlans}
-              />
+        <Container>
```

Comment:
Is this nested `<Container>` intentional? Does it accomplish something in particular?

Reply:
It's to be able to style the top nav differently from the body. Since changing to fluid means that all the body contents now went 100%. The nested is to keep them as they were.